### PR TITLE
docs(setup): Enhances desrcription of how to connect to UI

### PR DESCRIPTION
**Pull request by dorbin**

### CODE REVIEW

File: setup/install/deploy.md [hunk 1]
```diff
@@ -39,20 +39,27 @@ deploy apply`.
...
+1. Run the following command:
 
-```bash
-hal deploy connect
-```
+   ```bash
+   hal deploy connect
+   ```
+
+   If necessary, set up an SSH tunnel to the host running Halyard.
```
Comment (lwander) [inline setup/install/deploy.md:48]: I might swap this line and the one below

File: setup/install/deploy.md [hunk 2]
```diff
@@ -39,20 +39,27 @@ deploy apply`.
...
+
+1. Navigate to [localhost:9000](localhost:9000).
+
+### Alternatives
 
-If you want to make Spinnaker publicly reachable without running that command,
-please read the [Halyard FAQ](/setup/quickstart/faq/).
+* You can make Spinnaker publicly reachable without running this command,
+as described
+[here](/setup/quickstart/faq//#i-want-to-expose-localdebian-spinnaker-on-a-public-ip-address-but-it-always-binds-to-localhost)
```
Comment (lwander) [inline setup/install/deploy.md:59]: Worth mentioning in the link text that one is for localdebain, and the other is for k8s?

File: setup/install/deploy.md (hunk 1 above)
Comment (dorbin) [reply setup/install/deploy.md:48]: Yeah, I agonized over the order for a while, but I'd like to keep it like this. I feel like tunnelling just to run this command takes precedence over the (important but incidental) information that we're forwarding these ports.

File: setup/install/deploy.md (hunk 2 above)
Comment (dorbin) [reply setup/install/deploy.md:59]: Aye. Done.

File: setup/install/deploy.md (hunk 1 above)
Comment (lwander) [reply setup/install/deploy.md:48]: Ok, SG

**Review by lwander — APPROVED** [review]

**Pull request merged**